signatures/dilithium/dilithium-shake/impl/ref.yaml

```diff
@@ -0,0 +1,8 @@
+name: Reference
+side channel guarding:
+  timing: true
+  branching: true
+type: reference
+platform: c
+dependencies:
+  - OpenSSL
```

Comment:
seems they only use it for aes, so not here, right?

Reply:
I looked it up, they actually use OpenSSL only for KATs. I will remove it.